.github/workflows/ci.yml

```diff
@@ -145,7 +145,7 @@ jobs:
           command: udeps
           args: --locked --all-targets --all-features
 
-  cargo-build-wasm-example:
+  build-test-wasm-blobs:
```

Comment:
- This job just seems to build binaries and nothing else? 
- Now that we're building them on CI for e2e tests I think we can remove this `build-test-wasm-blobs` as it will just add CI time for no additional benefit.

Reply:
Oops, that's fair. This was left in when I assumed that files were persisted across each check.